miv/statistics/burst.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from miv.typing import SpikestampsType
+
+
+def burst(spiketrains: SpikestampsType, channel: float, min_isi: float, min_len: float):
+    """
+    Calculates parameters critical to characterize bursting phenomenon on a single channel
+    Bursting is defined as the occurence of a specified number of spikes (usually >10), with a small interspike interval (usually < 100ms) [1]_, [2]_
+
+    Parameters
+    ----------
+    spikes : SpikestampsType
+           Single spike-stamps
+    Channel : float
+       Channel to analyze
+    min_isi : float
+       Minimum Interspike Interval (in seconds) to be considered as bursting [standard = 0.1]
+    min_len : float
+       Minimum number of simultaneous spikes to be considered as bursting [standard = 10]
+
+    Returns
+    -------
+    start_time: float
+           The time instances when a burst starts
+    burst_duration: float
+           The time duration of a particular burst
+    burst_len: float
+            Number of spikes in a particular burst
+    burst_rate: float
+            firing rates corresponding to particular bursts
+
+    ..[1] Chiappalone, Michela, et al. "Burst detection algorithms for the analysis of spatio-temporal patterns
+    in cortical networks of neurons." Neurocomputing 65 (2005): 653-662.
+    ..[2] Eisenman, Lawrence N., et al. "Quantification of bursting and synchrony in cultured
+    hippocampal neurons." Journal of neurophysiology 114.2 (2015): 1059-1071.
+
+    """
+    spike_interval = np.diff(
+        spiketrains[channel].magnitude
+    )  ## Calculate Inter Spike Interval (ISI)
+    A = np.array(spike_interval)
+    B = np.array(spike_interval)
+    B[A > min_isi] = 0
```

Comment:
How about just `B = (spike_interval <= min_isi).astype(np._int)`? We can save `A` memory allocation.

Reply:
Thanks. Implemented as suggested

commit 1d9d31d5d93e5669b3519f324c9c968e67418cfa